packages/sanity/src/structure/i18n/resources.ts

```diff
@@ -95,6 +95,13 @@ const structureLocaleStrings = defineLocalesResources('structure', {
   'banners.deleted-document-banner.text': 'This document has been deleted.',
   /** The text content for the deprecated document type banner */
   'banners.deprecated-document-type-banner.text': 'This document type has been deprecated.',
+  /** The text for publish action for discarding the version */
+  'banners.live-edit-draft-banner.discard.tooltip': 'Discard draft',
+  /** The text for publish action for the draft banner */
+  'banners.live-edit-draft-banner.publish.tooltip': 'Publish to continue editing',
+  /** The text content for the live edit document when it's a draft */
+  'banners.live-edit-draft-banner.text':
+    'This live document cannot be edited while a draft version of it exists. Publish or discard the draft in order to continue live editing it.',
```

Comment:
I suggest that we provide slightly more details here:

```suggestion
The type <b>{{schemaType}}</b> has `liveEdit` enabled, but a draft version of this document exists. Publish or discard the draft in order to continue live editing it.',
```

I don't think it makes it much more confusing to non-devs, and we can probably assume they will forward the warning to someone that knows if they are unsure.

Reply:
Fixed the styles but done :)